### DISCUSSION TITLE
Understanding Blocked Reasons

### DISCUSSION THREAD

**Discussion by helenafranca** [GoogleChromeLabs/ps-analysis-tool#536] [Support Forum] [answered]
It is still not clear to me the differences between the blocked reasons. Can you please explain?

**Comment by amedina** [comment, marked as answer]
Cookies can be blocked due to a variety of reasons. There could be problems with some attributes of the cookie. It could be because the browser is already blocking 3P cookies as part of the facilitated period experiment (`
ThirdPartyPhaseout`), the cookie's domain did not match exactly the domain of the top-level site (`DomainMismatch`), and so on.
